src/data/markdown/translated-guides/en/02 Using k6/11 Environment variables.md

```diff
@@ -47,36 +56,52 @@ PS C:\k6> $env:MY_HOSTNAME="test.k6.io"; k6 run script.js
 
 </CodeGroup>
 
-or using an [`-e` / `--env` CLI flag](/using-k6/options#supply-environment-variables) (which will be the same for all platforms):
+> #### ⚠️ Warning
+>
+> By default, passing system environment variables does not work for `k6 archive`, `k6 cloud` and `k6 inspect`. It is a security measure to avoid the risk of uploading sensitive data to k6 Cloud. You can override this mode by explicitly specifying [`--include-system-env-vars`](https://k6.io/docs/using-k6/options/#include-system-env-vars).
 
-<CodeGroup labels={[]} lineNumbers={[true]}>
+</Collapsible>
+
+## Configure k6 options with environment variables
+
+k6 [options](/using-k6/options) can be configured by passing environment variables. Consider the following basic test script:
+
+```javascript
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function () {
+  const res = http.get('https://test.k6.io');
+  sleep(1);
+}
+```
+
+By default, running the above script locally will execute a single iteration using one virtual user(VU). We can modify the **default behavior** by passing along [k6 options](/using-k6/options) as environment variables. For example, we can configure the script to run 10 virtual users for a duration of 10 seconds:
+
+<CodeGroup labels={["Bash", "Windows: CMD", "Windows: PowerShell"]} lineNumbers={[false]}>
 
 ```bash
-$ k6 run -e MY_HOSTNAME=test.k6.io script.js
+$ K6_VUS=10 K6_DURATION=10s k6 run script.js
+```
+
+```bash
+C:\k6> set "K6_VUS=10 K6_DURATION=10s" && k6 run script.js
+```
+
+```bash
+PS C:\k6> $env:K6_VUS=10 ; $env:K6_DURATION="10s" ; k6 run script.js
 ```
 
 </CodeGroup>
 
-Note: This can *not* be used to configure k6 with environment variables as listed on the [options](/using-k6/options) page. In other words `-e K6_ITERATIONS=120` will *not* configure the script [iterations](/using-k6/options#iterations), it will just provide `__ENV.K6_ITERATIONS` to the script, unlike `K6_ITERATIONS=120 k6 run script.js`.
+As demonstrated above, you will need to prefix `K6_` in the environment variable name in order for k6 to evaluate it as an **option parameter**. However, be aware not all options are supported as environment variables. You can confirm by checking the documentation for each option.
 
-The environment variable could then be used as follows in a script:
+Note that when you use multiple ways to define options for a script, there's an [order of precedence](https://k6.io/docs/using-k6/options#using-options) that is used to determine which option is actually used. To ensure you are always working with the highest precedence, always use command-line flags instead of environment variables:
```

Comment:
```suggestion
Note that when you use multiple ways to define options for a script, there's an [order of precedence](/using-k6/options#using-options) that is used to determine which option is actually used. To ensure you are always working with the highest precedence, always use command-line flags instead of environment variables:
```

Reply:
Committed as suggested